sources/packages/web/src/composables/useFormatters.ts

```diff
@@ -304,6 +309,21 @@ export function useFormatters() {
     return `${negativeValueDisplay}$${Math.abs(currencyValue).toFixed(2)}`;
   };
 
+  /**
+   * Validates if the given date 1 is before the given date 2.
+   * @param date1 given date 1.
+   * @param date2 given date 2.
+   * @returns if the given date 1 is before the given date 2.
+   */
+  const isBeforeDate = (
```

Comment:
This util method looks good.  But why do we do the format here?
We have same method slightly more generic (without doing the format ) in the API side.  May be we can just use the same way. Let me know if I am missing something.

![image](https://github.com/bcgov/SIMS/assets/54600590/dc0febb9-971d-49d9-b4aa-3f33d0c4e7a7)

Reply:
without formatting date is a timestamp, here the comparison is only for date,
![image](https://github.com/bcgov/SIMS/assets/77353155/f5c553b6-bff7-4d74-b4f4-879ca76cc109)